Parsers/ASimNetworkSession/Parsers/ASimNetworkSessionMicrosoftSysmon.yaml

```diff
@@ -13,7 +13,7 @@ References:
 - Title: ASIM
```

Comment:
ASIM schema "NetworkSession" version "0.2.5" is incorrect. The correct version for ASIM schema "NetworkSession" is "0.2.6"

Reply:
done